### PR TITLE
updated with 2021 certs/endpoints

### PR DESCRIPTION
**Pull request by solipet**
Updates the Developer docs for the SAML 2021 certs/endpoints. Note that I split out the staging/production certs since in 2021 they are not the same.

### CODE REVIEW

File: _pages/saml.md [hunk 1]
```diff
@@ -55,86 +55,124 @@ Here are values needed to configure your service provider (SP) to work with logi
...
+vIeYwOGwc6q14kLHmq/ZBzbeZPEH2wPORwXouWOonxAWAHvrUUjt0uEY3y0VXRD5
+LnmMjyTlRLqBQyxJWEnJYeeWhIk0ARJMcerkaLcMASUlL6Sua7aI8E2qXOImJofC
+I+mn1PzJcSXvJE/Xi3UVt0GMVgIK7ZsKnAnDR6JulJOq1lkLh3QvERfXQJ4Zz88V
+lzdI/A8lml7s8FEL3jF/rd6lUg==
+-----END CERTIFICATE-----
+```
+  </div>
 </div>
 
 #### Annual Certificate Rotation
```
Comment (orenyk) [inline _pages/saml.md:178]: **Question:** Do we want to update this section as well to reference the 2021-2022 rotation?

**Review by orenyk — APPROVED** [review]
Looks good! One question about updating the rotation section but otherwise :shipit:

File: _pages/saml.md (hunk 1 above)
Comment (solipet) [reply _pages/saml.md:178]: Yes! Updated in e335cab. Thanks!

**Review by achapm — APPROVED** [review]
LGTM

**Pull request merged**